HKMP/Game/Client/Entity/MantisLord.cs

```diff
@@ -192,7 +233,8 @@ EntityType entityType
             // Determine the base state class state using the throne state
             var baseStateIndex = new byte?();
             if (stateIndex.HasValue) {
-                baseStateIndex = stateIndex == (byte) ThroneState.Empty ? (byte) State.Active : (byte) State.Idle;
+                baseStateIndex = (stateIndex == (byte) ThroneState.Empty ||
```

Comment:
This is getting quite large for a ternary operator. Moreover, it can be removed in favour of adding it in one of the if-statements below. Namely, one of the if-statements has the exact same condition, which is where you can set the value of `baseStateIndex` (and the other value in the, currently non-existant, `else` clause).